src/Makefile

```diff
@@ -1625,51 +1122,10 @@ wrappers_wazuh_o := $(wrappers_wazuh_c:.c=.o)
 wrappers_wazuh_os_crypto_c := $(wildcard unit_tests/wrappers/wazuh/os_crypto/*.c)
 wrappers_wazuh_os_crypto_o := $(wrappers_wazuh_os_crypto_c:.c=.o)
 
-wrappers_wazuh_os_execd_c := $(wildcard unit_tests/wrappers/wazuh/os_execd/*.c)
-wrappers_wazuh_os_execd_o := $(wrappers_wazuh_os_execd_c:.c=.o)
-
-wrappers_wazuh_os_net_c := $(wildcard unit_tests/wrappers/wazuh/os_net/*.c)
-wrappers_wazuh_os_net_o := $(wrappers_wazuh_os_net_c:.c=.o)
-
-wrappers_wazuh_os_regex_c := $(wildcard unit_tests/wrappers/wazuh/os_regex/*.c)
-wrappers_wazuh_os_regex_o := $(wrappers_wazuh_os_regex_c:.c=.o)
-
-wrappers_wazuh_os_xml_c := $(wildcard unit_tests/wrappers/wazuh/os_xml/*.c)
-wrappers_wazuh_os_xml_o := $(wrappers_wazuh_os_xml_c:.c=.o)
-
-wrappers_wazuh_shared_c := $(wildcard unit_tests/wrappers/wazuh/shared/*.c)
-wrappers_wazuh_shared_o := $(wrappers_wazuh_shared_c:.c=.o)
-
-wrappers_wazuh_syscheckd_c := $(wildcard unit_tests/wrappers/wazuh/syscheckd/*.c)
-wrappers_wazuh_syscheckd_o := $(wrappers_wazuh_syscheckd_c:.c=.o)
-
-wrappers_wazuh_wazuh_db_c := $(wildcard unit_tests/wrappers/wazuh/wazuh_db/*.c)
-wrappers_wazuh_wazuh_db_o := $(wrappers_wazuh_wazuh_db_c:.c=.o)
-
-wrappers_wazuh_wazuh_modules_c := $(wildcard unit_tests/wrappers/wazuh/wazuh_modules/*.c)
-wrappers_wazuh_wazuh_modules_o := $(wrappers_wazuh_wazuh_modules_c:.c=.o)
-
-wrappers_wazuh_monitord_c := $(wildcard unit_tests/wrappers/wazuh/monitord/*.c)
-wrappers_wazuh_monitord_o := $(wrappers_wazuh_monitord_c:.c=.o)
-
-wrappers_wazuh_os_auth_c := $(wildcard unit_tests/wrappers/wazuh/os_auth/*.c)
-wrappers_wazuh_os_auth_o := $(wrappers_wazuh_os_auth_c:.c=.o)
-
-wrappers_wazuh_addagent_c := $(wildcard unit_tests/wrappers/wazuh/addagent/*.c)
-wrappers_wazuh_addagent_o := $(wrappers_wazuh_addagent_c:.c=.o)
-
-wrappers_client_agent_c := $(wildcard unit_tests/wrappers/wazuh/client-agent/*.c)
-wrappers_client_agent_o := $(wrappers_client_agent_c:.c=.o)
-
-wrappers_wazuh_config_c := $(wildcard unit_tests/wrappers/wazuh/config/*.c)
-wrappers_wazuh_config_o := $(wrappers_wazuh_config_c:.c=.o)
 
 wrappers_data_provider_c := $(wildcard unit_tests/wrappers/wazuh/data_provider/*.c)
 wrappers_data_provider_o := $(wrappers_data_provider_c:.c=.o)
 
-wrappers_logcollector_c := $(wildcard unit_tests/wrappers/wazuh/logcollector/*.c)
-wrappers_logcollector_o := $(wrappers_logcollector_c:.c=.o)
-
 wrappers_windows_c := $(wildcard unit_tests/wrappers/windows/*.c)
 wrappers_windows_o := $(wrappers_windows_c:.c=.o)
```

Comment:
I understand that the Windows wrapper is no longer useful because it does not compile the Windows agent.
```suggestion
```